scripts/gas-refund-program/computeGasRefund.ts

```diff
@@ -87,7 +87,15 @@ async function startComputingGasRefundAllChains() {
 }
 
 startComputingGasRefundAllChains()
-  .then(() => process.exit(0))
+  .then(ps => {
+    const maybeOneRejected = ps.find(
+      p => p.status === 'rejected',
+    ) as PromiseRejectedResult;
```

Comment:
The right way to write a discriminant for TS is `(p): p is PromiseRejectedResult => ...`, then no need to cast

Reply:
if we'd use `Promise.all` instead of `Promise.allSettled` on line 76, wouldn't it give the same result, but without  the whole `ps.find` thing?

Reply:
ok @Velenir, I'll apply change as soon as I can

@alexshchur nop combination of promise.all + process.exit(1) would interrupt other promises